src/auditd-config.c

```diff
@@ -1961,7 +1996,9 @@ void free_config(struct daemon_conf *config)
         free((void *)config->krb5_key_file);
 	free((void *)config->plugin_dir);
         free((void *)config_dir);
-        free(config_file);
+        if (config_file != NULL)
+		free(config_file);
```

Comment:
Same comment here, free can handle NULL values.

Reply:
As before, I like to control memory management routines.

Reply:
The if has been removed.